state/app/DataManager.kt

```diff
@@ -71,17 +71,21 @@ class DataManager {
     inner class Connection {
 
         private val CONNECTION_SERVER = "connection.server"
-        private val CONNECTION_ADDRESS = "connection.address"
+        private val CONNECTION_CORE_ADDRESS = "connection.core_address"
+        private val CONNECTION_CLUSTER_ADDRESSES = "connection.cluster_addresses"
         private val CONNECTION_USERNAME = "connection.username"
         private val CONNECTION_TLS_ENABLED = "connection.tls_enabled"
         private val CONNECTION_CA_CERTIFICATE = "connection.ca_certificate"
 
         var server: Property.Server?
             get() = properties?.getProperty(CONNECTION_SERVER)?.let { Property.Server.of(it) }
             set(value) = value?.let { setProperty(CONNECTION_SERVER, it.displayName) } ?: Unit
-        var address: String?
-            get() = properties?.getProperty(CONNECTION_ADDRESS)
-            set(value) = value?.let { setProperty(CONNECTION_ADDRESS, it) } ?: Unit
+        var coreAddress: String?
+            get() = properties?.getProperty(CONNECTION_CORE_ADDRESS)
+            set(value) = value?.let { setProperty(CONNECTION_CORE_ADDRESS, it) } ?: Unit
+        var clusterAddresses: MutableList<String>?
+            get() = properties?.getProperty(CONNECTION_CLUSTER_ADDRESSES)?.split(",")?.toMutableList()
```

Comment:
Commas are not valid in IPv4, IPv6. There are some competing opinions about whether commas are valid in host names, but strictly 
> "[...] any binary string whatever can be used as the label of resource record."

https://www.rfc-editor.org/rfc/rfc2181

So there is no real valid delimiter that we can use here that is universally valid, to my understanding.